src/Search/Search.js

```diff
@@ -22,15 +18,12 @@ const Search = ({ filterMakeup }) => {
       <input
         className="search-bar"
         aria-label="search bar"
-        placeholder="Search..."
+        placeholder={`Search ${category.split('_').join(' ')}...`}
         name="searchInput"
         value={searchInput}
         onChange={(e) => handleChange(e)}
         autoComplete="off"
       />
-      <button className="clear-btn" type="submit" onClick={clearSearchInput}>
-        clear
-      </button>
     </form>
   );
 };
```

Comment:
Just putting Cypress notes here since those files aren't in the PR:
- add this to your scripts in `package.json`:
```
    "cypress": "./node_modules/.bin/cypress open"
```

- You said: "please look at my cypress issue card." I don't see any cypress tests in this branch...Am I missing something?
- "I also have links to different websites for more info in my footer. Do I need to test all of these?" I would test that the images/links appear on page load, but you don't need to test that the links work.

Reply:
I have questions on a cypress issue card. I don't have the tests written. I'm planning to start those now, just wanted to see if I needed to stub and test every navLink...?

Reply:
no - i would do ONE and make sure it's working. No need to do it for all of them